extensions/SemanticMediaWiki/DefaultSettings.php

```diff
@@ -148,14 +148,14 @@
 # move their content to other pages. If the namespace is not registered, then
 # existing pages in this namespace cannot be found in the wiki.
 ##
-if ( !isset( $GLOBALS['smwgHistoricTypeNamespace'] ) ) {
-	$GLOBALS['smwgHistoricTypeNamespace'] = false;
+if ( !isset( $smwDefaults['smwgHistoricTypeNamespace'] ) ) {
+	$smwDefaults['smwgHistoricTypeNamespace'] = false;
 }
 ##
 
 ###
 # If you already have custom namespaces on your site, insert
-#    $GLOBALS['smwgNamespaceIndex'] = ???;
+#    $smwgNamespaceIndex = ???;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)